dataset_config/1000_genomes/bigquery.json

```diff
@@ -21,40 +21,44 @@
+        "Chr 4 VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_4_vcf",
+        "Chr 5 VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_5_vcf",
+        "Chr 6 VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_6_vcf",
+        "Chr 7 VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_7_vcf",
+        "Chr 8 VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_8_vcf",
+        "Chr 9 VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_9_vcf",
+        "Chr 10 VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_10_vcf",
+        "Chr 11 VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_11_vcf",
+        "Chr 12 VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_12_vcf",
+        "Chr 13 VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_13_vcf",
+        "Chr 14 VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_14_vcf",
+        "Chr 15 VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_15_vcf",
+        "Chr 16 VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_16_vcf",
+        "Chr 17 VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_17_vcf",
+        "Chr 18 VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_18_vcf",
+        "Chr 19 VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_19_vcf",
+        "Chr 20 VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_20_vcf",
+        "Chr 21 VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_21_vcf",
+        "Chr 22 VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_22_vcf",
+        "Chr X VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_X_vcf",
+        "Chr Y VCF":"verily-public-data.human_genome_variants.1000_genomes_sample_info.chr_Y_vcf",
+        "WGS High Coverage BAM":"verily-public-data.human_genome_variants.1000_genomes_sample_info.wgs_high_cov_bam",
+        "WGS Low Coverage BAM":"verily-public-data.human_genome_variants.1000_genomes_sample_info.wgs_low_cov_mapped_bam",
+        "WGS High Coverage CRAM":"verily-public-data.human_genome_variants.1000_genomes_sample_info.wgs_high_cov_cram",
+        "WGS Low Coverage CRAM":"verily-public-data.human_genome_variants.1000_genomes_sample_info.wgs_low_cov_mapped_cram",
+        "Exome BAM":"verily-public-data.human_genome_variants.1000_genomes_sample_info.exome_mapped_bam",
+        "Exome CRAM":"verily-public-data.human_genome_variants.1000_genomes_sample_info.exome_mapped_cram",
+        "FASTQ":"verily-public-data.human_genome_variants.1000_genomes_sample_info.exome_cram"
```

Comment:
I ended up deleting people with fastq, delete this line

Reply:
Done.